pilgen/src/lib.rs

```diff
@@ -166,7 +167,7 @@ impl ASMPILConverter {
                 assert_eq!(self.pc_name, None);
                 self.pc_name = Some(name.to_string());
                 self.line_lookup
-                    .push((name.to_string(), "line".to_string()));
```

Comment:
Maybe `p_line`? IMO a `pc` is always a register or at least a runtime thing.

Reply:
I see, changed!